lib/spack/spack/cmd/audit.py

```diff
@@ -41,6 +41,14 @@ def setup_parser(subparser):
     # Audit package recipes
     pkg_parser = sp.add_parser("packages", help="audit package recipes")
 
+    # TODO: this should be set to true and removed as an option eventually
```

Comment:
(Nitpick) It might be better to add a target Spack version, so we are sure to revisit this at some point, when addressing a future release. For instance:
```suggestion
    # TODO (Spack v0.24): this should be set to true and removed as an option eventually
```
Saying this because I recently deleted some TODOs in code that were there for years :laughing:

Reply:
I just removed `--strict-variants` now that everything is fixed.